src/Bulk.php

```diff
@@ -133,8 +133,12 @@ public function getActions(): array
      */
     public function addDocument(Document $document, ?string $opType = null): self
     {
-        if (!$document->hasRetryOnConflict() && $this->_client->hasConnection() && $this->_client->getConnection()->hasParam('retryOnConflict') && ($retry = $this->_client->getConnection()->getParam('retryOnConflict')) > 0) {
-            $document->setRetryOnConflict($retry);
+        if (!$document->hasRetryOnConflict()) {
+            $retry = $this->_client->getConfigValue('retryOnConflict', 0);
```

Comment:
Is this directly related to the change? Maybe make it a quick second PR if not?

Reply:
Yes. because Client class doesn't have getConnection method so I had to find another way to get configuration value